tests/hawtio-test-suite/src/main/java/io/hawt/tests/features/pageobjects/pages/ConnectPage.java

```diff
@@ -24,7 +19,15 @@ public class ConnectPage extends HawtioPage {
 
     private static final By CONNECTION_LIST = By.id("connection-list");
 
+    private static final By CONNECTION_LOGIN_FORM = By.id("connect-login-form");
+
+    private static final By FOOTER_BUTTON = By.cssSelector("footer button.pf-m-primary");
+
     public void addConnection(String name, URL connection) {
+
+    if ($(CONNECTION_LIST).isDisplayed()) {
+        return;
```

Comment:
We should eventually double check that the connection name already exists and not expect that there will always be just one connection, you can just add a todo and explanation of why this workaround exists.